frontend/src/components/Playback/Autoplay.js

```diff
@@ -25,16 +26,23 @@ const AutoPlay = ({instruction, preloadMessage, onPreloadReady, playConfig, sect
         time.current = t;
     };
 
-
     // Handle view logic
     useEffect(() => {
         switch (state.view) {
-            case RECOGNIZE:                
+            case RECOGNIZE:
```

Comment:
You can get rid of this whole useEffect loop if there is only one state, right? That would be awesome!